certbot/certbot/_internal/ocsp.py

```diff
@@ -65,21 +64,14 @@ def ocsp_revoked(self, cert):
         .. todo:: Make this a non-blocking call
 
         :param `.storage.RenewableCert` cert: Certificate object
+
         :returns: True if revoked; False if valid or the check failed or cert is expired.
         :rtype: bool
 
         """
-        cert_path, chain_path = cert.cert, cert.chain
+        return self.ocsp_revoked_cert(cert.cert, cert.chain)
 
-        # Let's Encrypt doesn't update OCSP for expired certificates,
-        # so don't check OCSP if the cert is expired.
-        # https://github.com/certbot/certbot/issues/7152
-        now = pytz.UTC.fromutc(datetime.utcnow())
-        if cert.target_expiry <= now:
-            return False
-        return self.revoked(cert_path, chain_path)
-
-    def revoked(self, cert_path, chain_path, response_file=None):
+    def ocsp_revoked_cert(self, cert_path, chain_path, response_file=None):
```

Comment:
Oh yeah, this makes perfect sense, I indeed overlooked the actual class naming there and agree that it can be confusing, especially when we're moving this to the public API. I changed the name to ocsp_revoked_by_paths.